test/unit/spec/EnrollU2F_spec.js

```diff
@@ -103,7 +123,46 @@ function (Okta,
     });
 
     Expect.describe('Enroll factor', function () {
+      itp('shows error if wrong browser', function () {
+        delete window.u2f;
+        mockFirefox(false);
+
+        return setup().then(function (test) {
+          expect(test.form.errorHtml()).toHaveLength(1);
+          expect(test.form.errorHtml().html()).toEqual(Okta.loc('u2f.error.notSupportedBrowser', 'login'));
```

Comment:
If there is an error in Okta.loc, the tests for Okta.loc should fail.

I am not sure if it is a good idea when for changing texts we have to change not only the i18n, but also the tests.